README.md

```diff
@@ -61,3 +61,27 @@ groovy order.groovy bench
 cd typescript
 yarn run bench
 ```
+
+## Ethereum and Starknet Keys
+
+Ethereum (L1) private key is used to sign a signature. This signature is deterministic
+and can only be generated by the owner of the private key.
+
+This signature is then used to generate Starknet (L2) private key
+(also referred to as Paradex private key).
+
+Starknet (L2) private key can used to deduce Starknet public key.
+This private key is also used to sign auth, onboarding and order signatures.
```

Comment:
```suggestion
This private key is also used to sign auth, onboarding and order messages.
```
Another option is to say "used to generate auth, onboarding and order signatures"